agent/consul/leader.go

```diff
@@ -390,7 +402,7 @@ func (s *Server) initializeLegacyACL() error {
 		if err != nil {
 			return fmt.Errorf("failed to create anonymous token: %v", err)
 		}
-		s.logger.Printf("[INFO] acl: Created the anonymous token")
+		s.logger.Info("Created the anonymous token")
```

Comment:
```suggestion
		s.logger.Info("Created the anonymous ACL token")
```

Reply:
This will print out "agent.acl Info Create the anonymous ACL token", so this is already prefaced with ACL but I'm not opposed to adding this - just want to clarify

Reply:
Right. Nevermind.